core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance.jelly

```diff
@@ -0,0 +1,122 @@
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
+
+  <l:layout norefresh="true" type="full-screen" title="${it.displayName}">
+    <l:main-panel>
+      <link rel="stylesheet" href="${resURL}/css/google-fonts/roboto/css/roboto.css" type="text/css" />
+      <style type="text/css">
+        #configure-instance {
+          padding: 20px 100px 20px 100px;
+          margin: 8px;
+        }
+
+        #configure-instance form > div {
+          margin: 0 !important;
+        }
+
+        #configure-instance h1 {
+          font-family: 'roboto', sans-serif;
+          font-size: 48px;
+          margin-top: 30px;
+          font-weight: 500;
+        }
+
+        #configure-instance tr td {
+          padding-bottom: 2px;
+        }
+
+        #configure-instance tr td, #configure-instance input {
+          line-height: 25px;
+          margin-bottom: 6px;
+        }
+
+        #configure-instance input[type=text], #configure-instance input[type=password] {
+          border: 1px solid #ddd;
+          border-radius: 2px;
+          padding: 1px 8px;
+        }
+
+        #configure-instance tr.has-error label{
+          color: #CC0000;
+        }
+        
+        #configure-instance tr.has-error td input[type=text], 
+        #configure-instance tr.has-error td input[type=password],
+        #configure-instance tr.has-error td textarea {
+          border-color: #CC0000;
+          background-color: #fdeaea;
+        }
+        
+        #configure-instance tr td .help-text{
+          color: #999;
+          font-size: 12px;
+        }
+  
+        #configure-instance tr td .error-panel{
+          display: none;
+        }
+        #configure-instance tr.has-error td .error-panel{
+          display: block;
+        }
+      </style>
+
+     <div id="configure-instance">
+       <form action="${rootURL}/setupWizard/configureInstance" method="post">
+          <h1>${%Instance Configuration}</h1>
+          <div style="margin: 2em;">
+            <table>
+            <tr>
+              <td><label for="root-url">${%Jenkins URL}:</label></td>
+              <td>
+                <input type="text" name="rootUrl" id="root-url" class="root-url" value="" />
+                <div class="error-panel error" />
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <div class="help-text">${%jenkinsURL_help}</div>
+              </td>
+            </tr>
+            </table>
+          </div>
+         <script>
+            $('root-url').focus();
+            (function setInitialRootUrlFieldValue(){
+              var iframeUrl = window.location.href;
+              var iframeRelativeUrl = '/setupWizard/setupWizardConfigureInstance';
```

Comment:
Wrong! This strips the trailing slash, which is required. [JENKINS-51660](https://issues.jenkins-ci.org/browse/JENKINS-51660)